### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by josephwegner** [pushpop-project/pushpop#9]
I'm a big fan of Runscope's [RequestBin](http://requestb.in/) landing page - the branding is mostly RequestBin-related, with a Runscop plug at the bottom.

**Comment by hex337** [comment]
Yeah, its definitely clean. I like the idea of each project having its own brand, even if its under an umbrella. Gives it more cred.
